doc/under_sampling.rst

```diff
@@ -6,8 +6,25 @@ Under-sampling
 
 .. currentmodule:: imblearn.under_sampling
 
-You can refer to
-:ref:`sphx_glr_auto_examples_under-sampling_plot_comparison_under_sampling.py`.
+One way of handling imbalanced datasets is to reduce the number of observations from
+the majority class or classes. The most well known algorithm in this group is random
+undersampling, where samples from the majority classes are removed at random.
+
+But there are many other algorithms to help us reduce the number of observations in the
+dataset. These algorithms can be grouped based on their undersampling strategy into:
+
+- Prototype generation methods.
+- Prototype selection methods.
+
+And within the latter, we find:
+
+- Controlled undersampling
+- Cleaning methods
+
+We will discuss the different algorithms throughout this document.
+
+Refer to :ref:`sphx_glr_auto_examples_under-sampling_plot_comparison_under_sampling.py`
+for a comparison of the different undersampling methodologies.
```

Comment:
I don't think that we this sentence because sphinx will use the title which is already stating "Compare under-sampling samplers".

Reply:
Not sure I understand this suggestion, but let me try a fix

Reply:
OK, I got it.